.all-contributorsrc

```diff
@@ -13,6 +13,15 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "mohamilin",
+      "name": "Moh Amilin",
+      "avatar_url": "https://avatars.githubusercontent.com/u/50976240?v=4",
+      "profile": "https://www.linkedin.com/in/moh-amilin-6a5015128/",
+      "contributions": [
+        "doc"
+      ]
```

Comment:
Please revert the changes to this file. We shouldn't manually update this file.

We should use GitHub comments to add your name, like what I did here: https://github.com/SantriHub/app/pull/3#issuecomment-1153064029.
You can read more about it over here: https://allcontributors.org/docs/en/bot/usage